### DISCUSSION TITLE
What is the Fancy Tree?

### DISCUSSION THREAD

**Discussion by emyoulation** [PeWu/topola-viewer#113] [Q&A]
I'd like to expand wiki article on the [Interactive Family Tree add-on for Gramps](https://www.gramps-project.org/wiki/index.php/Addon:Interactive_Family_Tree) that feeds the Topola Genealogy Viewer. One of the View options is "Fancy Tree (Experimental)". Is there a basic description of the intent of this view variant?

![Addon-Interactive-Family-Tree-Topola-example-51.png](https://user-images.githubusercontent.com/69127217/189532329-9c998b76-16b3-4221-8259-7f03b142ad32.png)

**Comment by PeWu** [comment]
The "Fancy Tree" is a view inspired by this graphic: https://imgur.com/a/T02Kc7X

I never made it fully work, so it ended up in this "experimental" state. Nonetheless, I find it amusing for some family trees.

**Reply by emyoulation** [reply]
Thanks for the reply. Did some tweaks to [your addon's wiki page](https://gramps-project.org/wiki/index.php/Addon:Interactive_Family_Tree).   

Was wondering if you could take a look at the 3rd party Dashboard Gramplet by the [Isotammi project](https://github.com/Taapeli)? They took a different approach to a javascript and local server based graphing solution in Gramps. 
Kari's [Deep Connections Graph](https://github.com/Taapeli/isotammi-addons/tree/master/source/DeepConnectionsGraph) does a simple diagram of the connection between 2 persons in the Tree.

They left MUCH more of the local server control complexity visible in the Gramplet. (A bit TOO much.) And it is a weird place for it in the GUI.

PS. I find the current GUI location of the "Interactive Family Tree" (under the Tools -> Analysis and Exploration) is not intuitive.  There are similar interactive browser-based functionalities under the "Reports -> Web Pages".

Do you think your interactive viewer could live in Reports rather than Tools?
